### DISCUSSION TITLE
Showing logMagnitude on S11 instead of Smith chart

### DISCUSSION THREAD

**Discussion by iliavits** [jankae/LibreVNA#255] [General]
How can I show logMagnitude or Phase instead of Smith chart for S11 readings?
Thank you.

**Comment by jankae** [comment]
Did you check the user manual already?

**Reply by iliavits** [reply]
Yes, I have looked at the manual. I assume the data comes as a complex number, so I have tried to calculate the magnitude with math expression, but still it is displayed on top of Smith chart.

**Reply by jankae** [reply]
Yes, a smith chart is unable to show the data in any format other than as a smith chart. But have you tried adding S11 to one of the other graphs? Or removing the smith chart and use an XY-plot in its place instead?

No need to do any calculations on your own, the GUI handles all that for you. Please check chapter 6 of the user manual.

**Comment by iliavits** [comment]
Thanks, I think I have fixed it now.

**Comment by jankae** [comment]
okay, great :) I really feel like this is a problem that could be solved by just reading the user manual but in case someone else comes across this: the solution is to drag-and-drop the S11 trace on an XY-plot (one of the two graphs that show S12 and S21 by default)
